### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell v2.0.dev`  
`codespell -q 3 -L ans,fo,hist,inout,ist,ith,nd,sinc,uint -S ./volk`

### CODE REVIEW

**Comment by luzpaz** [comment]
Not sure where the formatting error is that makes github-action fail

File: grc/gui/ParamWidgets.py [hunk 1]
```diff
@@ -361,7 +361,7 @@ def _handle_clicked(self, widget=None):
         dir_dialog.set_local_only(True)
         dir_dialog.set_select_multiple(False)
         
-        # Show dialog and update paramter on success
+        # Show dialog and update parame:wqter on success
```
Comment (mormj) [inline grc/gui/ParamWidgets.py:364]: this one doesn't look right

Comment (luzpaz) [reply grc/gui/ParamWidgets.py:364]: :facepalm:  
will fix..thanks

Comment (luzpaz) [reply grc/gui/ParamWidgets.py]: Using vim sucks sometimes :neutral_face:  
Fixed in 2ff5af5

**Comment by mormj** [comment]
A couple of things to get CI to pass
1) run clang-format on the affected files - the build step above (Details) will show you which files are failing.  Likely it is whitespace differences.  If clang-format is set up correctly on your machine, you should be able to do this on the whole tree:
```
josh@josh-dev:/share/gnuradio/grdev/src/gnuradio$ find ./ -iname *.h | xargs clang-format -i
josh@josh-dev:/share/gnuradio/grdev/src/gnuradio$ find ./ -iname *.cc | xargs clang-format -i
```
2) Update the python binding hashes.  There is a simple mechanism to ensure that the python bindings stay in sync with the public c++ header files, by putting an md5sum in the binding file that needs to be updated when the .h file changes
For example, in include/gnuradio/block.h, run md5sum [PATH_TO]/gnuradio-runtime/include/gnuradio/block.h
Copy and paste this value into python/bindings/block_python.cc 
Do this for all affected blocks

If actual functionality were being changed, you would want to update the code in the bindings, but since it is just the comments, the hash can be updated without much thought.

**Comment by luzpaz** [comment]
@mormj thanks. I completed step 1. But I'm lost in regards to step 2.   
How can I find out what blocks are affected that I need to do this for ?

**Comment by mormj** [comment]
Any file where you touched the include/<module>/.h and there is a corresponding _python.cc in the python/bindings directory

Alternatively you can make clean on your local machine, and cmake will lead the way.  (may or may not have to make clean if you just run cmake - it will choke on every file where they don't match)

**Comment by luzpaz** [comment]
I was able to generate them with a quick one-liner:  
`git show --pretty="" --name-only c7e6496b9b5b72 | grep -e '\.h$' | while read file; do md5sum ${file}; done`

```
ee3b6013487881ab1dac5105e9fd55c7  gnuradio-runtime/include/gnuradio/block.h
3ab08834070348db57c720795b8de826  gnuradio-runtime/include/gnuradio/flowgraph.h
1664568abb8b7440216e8565354ffd7f  gr-blocks/include/gnuradio/blocks/phase_shift.h
143913f78329444b1cdda20cbe1d7a7c  gr-blocks/include/gnuradio/blocks/sample_and_hold.h
50555a585be0d02b5c3eddc195f8b018  gr-blocks/include/gnuradio/blocks/threshold_ff.h
3d7ebc78bf6a6c712f19f13f3e20648a  gr-digital/include/gnuradio/digital/cma_equalizer_cc.h
a4bdcaf2db30bfe32e518f4f4b62bafa  gr-digital/include/gnuradio/digital/constellation.h
a6425f9904862a6d1eac5e8f884e14cd  gr-digital/include/gnuradio/digital/symbol_sync_cc.h
be3b845025900d5a510daf14508add97  gr-digital/include/gnuradio/digital/symbol_sync_ff.h
c39c802e82dc6f53986dccca5ac67c7b  gr-digital/lib/clock_tracking_loop.h
0652e89ad8f24d5a259d3dac80522f31  gr-digital/lib/constellation_receiver_cb_impl.h
1ba01d1de09a956941a536ec238d5c3d  gr-qtgui/include/gnuradio/qtgui/freq_sink_c.h
5205d8f644420b78f2892542aad86aba  gr-qtgui/include/gnuradio/qtgui/freq_sink_f.h
6057d9100a92ed9a2ef5cdba54c0cdca  gr-qtgui/include/gnuradio/qtgui/plot_raster.h
78046dc7b0c34e92ec6d1aff9e0ab0c8  gr-qtgui/include/gnuradio/qtgui/plot_waterfall.h
a090f45dcc320d7e1f19f63c585e5aa0  gr-uhd/include/gnuradio/uhd/rfnoc_block.h
635a84f92d7ce031bcbe6bd4756ad28e  gr-uhd/include/gnuradio/uhd/rfnoc_rx_radio.h
6a7fea399cb9d3f5ac2849fb0d8135e4  gr-uhd/include/gnuradio/uhd/rfnoc_tx_radio.h
```

BUT...I'm having a dickens of a time figuring out where to put the hashes :disappointed:

**Comment by mormj** [comment]
For each of these, find the corresponding _python.cc file, somewhere in a python/bindings directory.

For example, the hash goes in gr-blocks/python/blocks/bindings/threshold_ff_python.cc for gr-blocks/include/gnuradio/blocks/threshold_ff.h

**Comment by luzpaz** [comment]
Just wondering, @mormj do y'all have a script that takes care of this?

**Comment by mormj** [comment]
Sort of - but not to just update the hashes.  Using bind_intree_file.py is still pretty manual.  Any automation you recommend (or feel like submitting) is helpful.

**Review by mormj — APPROVED** [review]

**Pull request merged**